pkg/result/processors/nolint_filter.go

```diff
@@ -227,11 +227,12 @@ func (p *NolintFilter) extractInlineRangeFromComment(text string, g ast.Node, fs
 		return buildRange(nil) // ignore all linters
 	}
 
+	text = strings.Split(text, "//")[0] // allow another comment after this comment
```

Comment:
If you want to avoid creating the slice of strings altogether, you could use [strings.Cut](https://pkg.go.dev/strings#Cut) here:

```suggestion
	text, _, _ = strings.Cut(text, "//") // allow another comment after this comment
```